docs/pages/deployment/configuration.md

```diff
@@ -0,0 +1,52 @@
+# Configuration
+
+## Environment variables
+
+### `LIVE_COMPOSITOR_API_PORT`
+
+API port. Defaults to 8001.
+
+### `LIVE_COMPOSITOR_OUTPUT_FRAMERATE`
+
+Output framerate for all output streams. This value can be a number or string in the `NUM/DEN` format , where both `NUM` and `DEN` are unsigned integers.
+
+### `LIVE_COMPOSITOR_STREAM_FALLBACK_TIMEOUT_MS`
+
+A timeout that defines when the compositor should switch to fallback on the input stream that stopped sending frames.
+
+### `LIVE_COMPOSITOR_LOGGER_LEVEL`
+
+Logger level. Value can be defined as `error`/`warn`/`info`/`debug`/`trace`.
+
+This value also supports syntax for more detailed configuration. See [`tracing-subscriber` crate documentation](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#example-syntax) for more info.
+
+### `LIVE_COMPOSITOR_LOGGER_FORMAT`
+
+Logger format. Supported options:
+- `json`
+- `compact`
+- `pretty`
+
+:::warning
+This option does not apply to logs produced by `FFmpeg` or the embedded Chromium instance used for web rendering.
+:::
```

Comment:
what does `:::` do?

Reply:
<img width="675" alt="image" src="https://github.com/membraneframework/video_compositor/assets/16838075/3888b22e-face-4bf2-8650-27467f2a681c">

^ This